src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,75 @@
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+
+type Props = {
+  person: Person[]
+  isLoading: boolean
+};
+export const PeopleTable: React.FC<Props> = ({ person, isLoading }) => {
+  const { slug } = useParams();
+
+  return isLoading ? <Loader /> : (
+    <>
```

Comment:
```suggestion
  if(isLoading) {
   return <Loader />
  }

  return (
    <>
    ...
```
suggestion to you, in my horrible opinion, looks better